cmd/agent/main.go

```diff
@@ -25,6 +26,7 @@ var (
 	addr            = "127.0.0.1:39000"
 	dataDir         = "./var/data"
 	logDir          = "./var/log"
+	logTrashDir     = "./var/log/trash"
```

Comment:
but that is a problem that user must use a input like "--log-trash=./var/log_trash", i think it has no need. so i prefer to use logDir/trash to do this.